inception-project/inception
# #4557 - Bulk process page should not show processes from other projects

**What's in the PR**
- Distinguish between ProjectTask and other tasks using marker interface
- Route user tasks and project tasks separately in SchedulerWebsockerControllerImpl
- Allow TaskMonitoringPanel to be configured more flexibly for the the Websocket topics it uses
- Moved logging UI code out to a new module
- Bit of cleaning up here and there

**How to test manually**
* Try running bulk processing jobs from different projects and users and see if they appear in the other projects/users

**Automatic testing**

**Documentation**
